sdk/eventhub/Azure.Messaging.EventHubs/assets/functions/help-functions.ps1

```diff
@@ -0,0 +1,115 @@
+
+function DisplayHelp
+{
+  <#
+    .SYNOPSIS
+      Displays the usage help text.
+
+    .DESCRIPTION
+      Displays the help text for usage.
+
+    .OUTPUTS
+      Help text for usage to the console window.
+  #>
+
+  $indent = "    "
+
+  Write-Host "`n"
```

Comment:
I'm not sure that I agree with breaking this out.  There's a very strong association between the help text and the script itself.   It may be better to keep these in the script for locality; those tweaking the script are less likely to forget about the help text if its staring them in the face.